config/routes.rb

```diff
@@ -28,11 +28,20 @@
   resources :solr_documents, only: [:show], path: '/catalog', controller: 'catalog'
 
   mount Spotlight::Engine, at: '/'
+  concern :exportable, Blacklight::Routes::Exportable.new
 
   resources :exhibits, path: '/', only: [] do
     resource :dor_harvester, controller: :"dor_harvester", only: [:create, :update]
     resource :bibliography_resources, only: [:create, :update]
     resource :viewers, only: [:create, :edit, :update]
+
+    resources :solr_documents, except: [:index], path: '/catalog', controller: 'catalog' do
+      concerns :exportable
```

Comment:
I'm a little surprised we need to re-mix-in `exportable`. What do you see without it?

Reply:
`No route matches..` and `undefined method `exhibit_root_path' for #<Spotlight::CatalogController:0x007ff41d7be2b0> Did you mean? exhibit_viewers_path`